includes/acl/compression/impl/track_error.impl.h

```diff
@@ -641,6 +641,8 @@ namespace acl
 		return calculate_transform_track_error(allocator, args);
 	}
 
+#ifndef ACL_COMPRESSION_OPTIMIZED
```

Comment:
Why strip this?

Reply:
They were unused.  They also didn't seem to make much sense, such as _"Calculates the worst compression error between two raw track arrays."_  I wouldn't expect raw track arrays to have any compression at all, never mind compression error.  It was part of my initial review of the compression system, but I can put them back if you prefer.